.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,3 +18,13 @@ If this pull request addresses a Help Wanted ticket, please link the relevant Gi
 
 Otherwise, link the JIRA ticket.
 -->
+
+
+#### Release Note
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your past-tense release note in the block below.
```

Comment:
What happens if one is unsure how to write a release note?

Reply:
any mantainer can re-write

Reply:
What is the process a community member should follow if the are unsure? Can we document that process?

Reply:
Yes, it will be included in the developer docs.